packages/grid/src/vaadin-grid-mixin.d.ts

```diff
@@ -98,6 +98,11 @@ export type GridDataProviderChangedEvent<TItem> = CustomEvent<{ value: GridDataP
  */
 export type GridExpandedItemsChangedEvent<TItem> = CustomEvent<{ value: TItem[] }>;
 
+/**
+ * Fired when the user selects or deselects an item.
+ */
+export type GridItemToggleEvent<TItem> = CustomEvent<{ item: TItem; selected: boolean; shiftKey: boolean }>;
```

Comment:
Should the event be fired from the grid or the selection column? Grid seems more natural, but on the other hand this event can never be fired without using the selection column and it's tied to its logic. Alternatively we could explicitly mention in the JSDoc that this will only be fired when using the selection column, the `isItemSelectable` property for example has similar remarks.

Reply:
Updated the JSDoc.